cli.js

```diff
@@ -411,30 +441,95 @@ function toDecimals(value, decimals, fixed) {
   return value
 }
 
+// List fetched from https://github.com/ethereum-lists/chains/blob/master/_data/chains
+function getExplorerLink() {
+  switch (netId) {
+    case 56:
+      return 'bscscan.com'
+    case 100:
+      return 'blockscout.com/poa/xdai'
+    case 137:
+      return 'polygonscan.com'
+    case 42161:
+      return 'arbiscan.io'
+    case 43114:
+      return 'snowtrace.io'
+    case 5:
+      return 'goerli.etherscan.io'
+    case 42:
+      return 'kovan.etherscan.io'
+    default:
+      return 'etherscan.io'
+  }
+}
+
+// List fetched from https://github.com/trustwallet/assets/tree/master/blockchains
 function getCurrentNetworkName() {
   switch (netId) {
     case 1:
-      return ''
+      return 'Ethereum'
+    case 56:
+      return 'BinanceSmartChain'
+    case 100:
+      return 'xDai'
+    case 137:
+      return 'Polygon'
+    case 42161:
+      return 'Arbitrum'
+    case 43114:
+      return 'Avalanche'
     case 5:
-      return 'goerli.'
+      return 'Goerli'
     case 42:
-      return 'kovan.'
+      return 'Kovan'
+    default:
+      return 'localRPC'
   }
 }
 
-function gasPrices(value = 80) {
+function getCurrentNetworkSymbol() {
+  switch (netId) {
+    case 56:
+      return 'BNB'
+    case 100:
+      return 'DAI'
+    case 137:
+      return 'MATIC'
+    case 43114:
+      return 'AVAX'
+    default:
+      return 'ETH'
+  }
+}
+
+function gasPricesETH(value = 80) {
   const tenPercent = (Number(value) * 5) / 100
   const max = Math.max(tenPercent, 3)
   const bumped = Math.floor(Number(value) + max)
   return toHex(toWei(bumped.toString(), 'gwei'))
 }
 
+function gasPrices(value = 5) {
+  return toHex(toWei(value.toString(), 'gwei'))
+}
+
 async function fetchGasPrice() {
   try {
-    const oracle = new GasPriceOracle()
-    const gas = await oracle.gasPrices()
-
-    return gasPrices(gas.fast)
+    const options = {
+      chainId: netId
+    }
+    // Bump fees for Ethereum network
+    if (netId == 1) {
+      const oracle = new GasPriceOracle(options)
+      const gas = await oracle.gasPrices()
+      return gasPricesETH(gas.instant)
```

Comment:
Unsupported networks by https://github.com/peppersec/gas-price-oracle will now return `1 gwei` as gasprices. Supports bumping fees for congested Ethereum network since only `BaseFee` will be deducted by the user account according to `EIP-1559`